src/System.Runtime.Serialization.Json/tests/ReflectionOnly/Resources/System.Runtime.Serialization.Json.ReflectionOnly.Tests.rd.xml

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<Directives xmlns="http://schemas.microsoft.com/netfx/2013/01/metadata">
+  <Library Name="*System.Private.DataContractSerialization*">
+    <Assembly Name="System.Private.DataContractSerialization" Dynamic="Required All">
```

Comment:
@shmao That's what i did first , that made sure about 30 tests passed .. then rest were failing with Arg_null exception ... Feel free to open an issue to investigate further and fix the test. Keeping the metadata for the whole assembly may not be necessary , but trying to figure out what to keep test by test is not efficient and very time consuming.